modules/battery_manager/test/TestPlan.md

```diff
@@ -89,15 +88,13 @@ Test Battery.
 
 **Testing edit (activate/deactivate/add) functionality**
   1. Check that you cannot edit an entry without filling out the required fields: `Instrument`, `Minimum age (days)`, `Maximum age (days)`, `Stage`.
-  2. Check that you can only enter a site that exists.
-  3. Check that you can only enter numbers between 0 and 99999 in Minumum age (days) and Maximum age (days).
-  4. Check that you can only enter numbers between 0 and 127 in Instrument order.
```

Comment:
not sure for what this is useful ? @HenriRabalais ?

Reply:
I don't think I wrote this, so I'm not sure why these limitations are imposed.

Reply:
maybe but that wouldnt be the place for it. @laemtl you can remove this line as well

Reply:
To be clear, which line should be removed?

Reply:
4. Check that you can only enter numbers between 0 and 127 in Instrument order.